src/NAV_SECTIONS.ts

```diff
@@ -1,21 +1,19 @@
-export type SECTION_LEAF_PAGE = {
+export interface SECTION_LEAF_PAGE {
   name: string;
   link: string;
-};
+}
 
-export type SECTION_PAGE = {
+export interface SECTION_PAGE {
   name: string;
-  pages: Array<SECTION_PAGE | SECTION_LEAF_PAGE>;
-};
+  pages: (SECTION_PAGE | SECTION_LEAF_PAGE)[];
```

Comment:
This was auto-fixed by [`@typescript-eslint/array-type`](https://typescript-eslint.io/rules/array-type), which is set to prefer the simpler shorthand type. If you prefer types you can change it to prefer the longer generic type and run `npm run lint -- --fix` again.